confluent-kafka-javascript-basic/README.md

```diff
@@ -0,0 +1,106 @@
+# Use confluent-kafka-javascript to produce and consume records
+
+This is a basic example of using the [confluent-kafka-javascript](https://github.com/confluentinc/confluent-kafka-javascript) client (early access as of when this was written, 4/29/24). 
+
+In this example, you'll produce database metadata records (made in the example with faker.js) to a Kafka topic and consume the records from the same topic. 
+
+![CC UI image](/graph.png)
+
+## Step 1: Clone and install
+
+`git clone https://github.com/Cerchie/basic-confluent-kafka-javascript-example.git` 
+
+`npm install` 
+
+## Step 2: Get set up in Confluent Cloud
+
+Sign up for [Confluent Cloud](https://www.confluent.io/confluent-cloud). 
+
+### To add an environment:
```

Comment:
optionally consider using the cloud quickstart CLI plugin to skip the env / cluster / credentials creation: not sure if it'd "just work" if you pass `--client nodejs` though